tools/Dashboard/DevHome.Dashboard/Views/AddWidgetDialog.xaml.cs

```diff
@@ -336,4 +336,21 @@ private void WidgetCatalog_WidgetDefinitionDeleted(WidgetCatalog sender, WidgetD
             }
         });
     }
+
+    private void ContentDialog_SizeChanged(object sender, SizeChangedEventArgs e)
+    {
+        const int ContentDialogMaxHeight = 684;
+
+        if (this.ActualHeight < ContentDialogMaxHeight)
+        {
+            AddWidgetNavigationView.Height = this.ActualHeight - AddWidgetTitleBar.ActualHeight;

```

Comment:
nit: Replace `if/esle` with `Math.Min`  (same comment for the other method)
```suggestion
            AddWidgetNavigationView.Height = Math.Min(this.ActualHeight, ContentDialogMaxHeight) - AddWidgetTitleBar.ActualHeight;
```